src/main/java/build/buf/protovalidate/EvaluatorBuilder.java

```diff
@@ -51,6 +51,7 @@ class EvaluatorBuilder {
 
   private final Env env;
   private final boolean disableLazy;
+  private final List<Descriptor> descriptors;
```

Comment:
This only looks to be referenced in the constructor. We should be able to remove this field and just iterate over the constructor arg below.